src/roadmap/release-schedule.md

```diff
@@ -1,95 +1,82 @@
-next minor or major release.
+Deis releases are not feature-based, in that dates are not linked to specific features. If a
+feature is merged before the release date, it is included in the next minor or major release.
 
-The master `git` branch of Deis always works. Only changes considered ready to
-be released publicly are merged, and releases are made from master.
+The master `git` branch of Deis should always work. Only changes considered ready to be released
+publicly are merged, and non-patch releases are made from master.
 
 ## Semantic Versioning
 
 Deis releases comply with [semantic versioning][], with the "public API" broadly
 defined as:
 
-- the REST API for *deis-controller*
-- etcd keys and values that are publicly documented
-- `deis` commands and options
-- essential `Makefile` targets
-- provider scripts under `contrib/`
+- the REST API for [the Controller][controller]
+- `deis` CLI commands and options
 
-Users of Deis can be confident that upgrading to a patch or to a minor release
-will not change the behavior of these items in a backward-incompatible way.
+Users of Workflow can be confident that upgrading to a patch or to a minor release will not change
+the behavior of these items in a backward-incompatible way.
 
 ## Release Criteria
 
-For any Deis release to be made publicly available, it must meet at least
-these criteria:
+For any WOrkflow release to be made publicly available, it must meet at least these criteria:
 
 - Passes all tests on the supported, load-balancing cloud providers
 - Has no new regressions in behavior that are not considered trivial
 
 ## Patch Releases
 
-A patch release of Deis includes backwards-compatible bug fixes. Upgrading to
-this version is safe and can be done in-place.
+A patch release of Workflow includes backwards-compatible bug fixes. Upgrading to this version is
+safe and can be done in-place.
 
-Backwards-compatible bug fixes to Deis are merged into the master branch at any
-time after they have [two approval comments][merge approval].
+Backwards-compatible bug fixes to Workflow are merged into the master branch at any time after they
+have [two approval comments][merge approval].
 
-Patch releases are created as often as needed, based on the priority of one or
-more bug fixes that have been merged. If time or severity is crucial, an
-individual maintainer can create a patch release without consensus from others.
-Patch releases are created from a previous release by cherry-picking specific
-bug fixes from the master branch, then applying and pushing the new release tag.
+Patch releases are created as often as needed, based on the priority of one or more bug fixes that
+have been merged. If time or severity is crucial, an individual maintainer can create a patch
+release without consensus from others. Patch releases are created from a previous release by
+cherry-picking specific bug fixes from the master branch, then applying and pushing the new release
```

Comment:
My understanding is the same as @krancour's and I think the language here mostly agrees.